src/%IPM/ResourceProcessor/Default/LocalizedMessages.cls

```diff
@@ -60,10 +60,29 @@ Method OnConfigureMappings(ByRef pParams) As %Status
 		Set tGlobalScope = ..ResourceReference.Module.GlobalScope && '$Get(pParams("Reload","ForceLocalScope"),0)
 		If 'tGlobalScope {
 			Set tSourceDB = ##class(%IPM.Utils.Module).GetRoutineDatabase($Namespace)
-			Set tSC = ##class(%IPM.Utils.Module).AddRoutineMapping($namespace,tName,,tSourceDB)
+			// Check if current file exists in the the routine database of the current namespace exists in current namespace('s database)
```

Comment:
Again use %SYS.Namespaces methods for this

Reply:
Created https://usjira.iscinternal.com/browse/HSIEO-6334 to handle this